contracts/protocol/facets/CustodyVault.sol

```diff
@@ -33,7 +33,6 @@ contract CustodyVaultFacet is Context, CustodianVaultErrors, Access, Custody, IF
      *
      * Reverts if:
      * - Custody region is paused
-     * - Caller is not the F-NFT contract owning the token
```

Comment:
To be precise, the auditors asked to remove this line from natspect for the internal `setupCustodianOfferVault` (line 562).

But now, looking at it again, I'm not even sure that they are completely correct - they say that the revert is "imposed by parent call chains", but in reality, the check happens only after `Custody.addItemToCustodianOfferVault` is called, which is inside `setupCustodianOfferVault`. 
So, maybe we dismiss the finding #487 ?